components/infra-proxy-service/server/proxy.go

```diff
@@ -160,6 +162,19 @@ func GetOrgAdminKeyFrom(secret *secrets.Secret) string {
 	return adminKey
 }
 
+// GetWebuiKeyFrom returns Webuikey
```

Comment:
The code is already written `GetOrgAdminKeyFrom`
We need to do this:
- Refactor the code to a common function
- Use the common function in both these functions